climpred/classes.py

```diff
@@ -84,20 +88,119 @@ class PredictionEnsemble:
     should house functions that both ensemble types can use.
     """
 
+    # ---------------
+    # Magic Functions
+    # ---------------
     @is_xarray(1)
     def __init__(self, xobj):
         if isinstance(xobj, xr.DataArray):
             # makes applying prediction functions easier, etc.
             xobj = xobj.to_dataset()
         has_dims(xobj, ['init', 'lead'], 'PredictionEnsemble')
-        self.initialized = xobj
-        self.uninitialized = {}
+        self._datasets = {'initialized': xobj}
+        # Reserve sub-dictionary for an uninitialized run.
+        self._datasets.update({'uninitialized': {}})
```

Comment:
Would it be better if `self._datasets = {'initialized': xobj, 'uninitialized': {}}` or is it clearer the way it is now?